OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        Madḫāne ʿĀlam, ending with a condemnation. Among the objects is also a drawing on parchment.</desc>
                                     <q xml:lang="am">ኍልቈ፡ ንዋዮ፡ ቅድሳት፡ ዘመድኃኔ፡ ዓለም፡
                                         <persName ref="PRS12911KidanaK"><roleName type="title">አባ፡</roleName> ኪዳነ፡ ክርስቶስ፡</persName> የሰጡ፡
                                     ፫መስቀል፡ ፫ጽና፡ ፩ናስ፡ የደም፡ ጽዋ፡
                                     ፩ጠርሙስ፡ ፩ቃጭል፡ ፩የናስ፡ ማብርያ፡
                                     ፩የፍም፡ መጫርያ፡ መረዋት፡
                                     ፩ጁህ፡ በርኖስ፡ ፩ማኅፈድ፡ ፩<ref type="work" corresp="LIT1560Gospel">ወንጌል፡</ref>
-                                        ፩<ref type="work" corresp="LIT1960Mashaf">ቅዳሴ፡</ref> ፩<ref type="work" corresp="LIT2384Taamme">ተአምረ፡ እግዝእትነ፡ ማርያም</ref>፡ ፩መዝገብ፡ ፩የብራና፡ ሥዕል፡
+                                        ፩<ref type="work" corresp="LIT1960Mashaf">ቅዳሴ፡</ref> ፩<ref type="work" corresp="LIT2384Taamme">ተአምረ፡
+                                            እግዝእትነ፡ ማርያም</ref>፡ ፩መዝገብ፡ 
+                                        ፩የብራና፡ ሥዕል፡
                                     ፪ቤት፡ ያሹሩት፡ በድሙሩ፡ ፳፩ነው፡
                                     ይህነን፡ የተጻፈውን፡ የሸጠና፡ የሰረቀ፡ የፋቀ፡ በሥልጣነ፡ ጴጥሮ<add place="above">ስ</add>፡ ወጳውሎስ፡
                                     ውጉዘ፡ ይኩንx x x
                                     </q>
                                 </item>
                                 
-                                <item xml:id="a2">
+                                <item xml:id="a2" corresp="#h2">
                                     <locus target="#5r"/>
-                                    <desc type="Inventory">An additional list of objects, separated from
+                                    <desc type="Inventory">An additional list of objects written in the same hand as <ref target="#a1"/>, separated from
                                     <ref target="#a1"/> by a fine line.</desc>
                                     <q xml:lang="am">በ፯ወቄት፡ ከ፫ድሪም፡ የጕልላት፡ መስቀል፡ የተሰራ፡ ቍጽሩ፡
                                     ፲፱ነው፡ ከቃቤት፡ አለ፡</q>
                                 </item>
                                 
                                 <item xml:id="a3">
                                     <locus target="#7r"/>
-                                    <desc type="OwnershipNote">Note written in pale black ink in the upper corner of the folio, stating that this manuscript belonged to 
+                                    <desc type="OwnershipNote">Note written in pale black ink in the upper inner corner of the folio, stating that this manuscript belonged to 
                                         <placeName ref="INS0101MadhaneAlam"/>.</desc>
                                     <q xml:lang="gez">ተአምረ፡ ማርያም፡ ዘቅዱስ፡ <placeName ref="INS0101MadhaneAlam">መድኃኔ፡ ዓለም፡</placeName></q>
                                 </item>
+                                
+                                <item xml:id="e1">
```

Comment:
We do create an extra item for each pen trial; Or in this case because the word divider is missing?